### DISCUSSION TITLE
Debris Heartbeat Wi-Fi Tarama Fikri

### DISCUSSION THREAD

**Discussion by mustafadilmac** [acikkaynak/deprem-yardim-projesi#64] [Ideas]
Aklıma gelen bir fikri paylaşmak istiyorum. Telefonlar ve aktif cihazlar otomatik wifi katılım desteği bulunuyor. hali hazırda wifi erişim noktasına bağlanmış olan cihazlar aynı isimdeki bir wifiye otomatik olarak katılabiliyor. 

![Group 213](https://user-images.githubusercontent.com/37265671/218263192-b75f395a-58b4-405a-8c8f-e948897cec05.jpg)

Deprem altında kalan insanların ilk iletişim eksikliklerinde bir dağıtıcı ile birden fazla network adı dağıtılabildiğini biliyoruz. Bu özellikle daha önce toplu alanlarda kullanılmış olan wifi isimleri şifreleri ile paket haline getirerek AFAD tarafından yayınlanarak wireshark gibi bir ara izleme yazılımı ile network istekleri taranabilir. Otomatik işlemler(güncelleme, bildirim vb.) gibi işlemler dışında mesaj gönderme mail atma gibi olaylar filtrelendiği takdirde burada bir hayat belirtisi gözlemlenebilir. Üstelik macos'ta keychaindeki tüm SSID ve şifreler paylaşılabiliyor. Yardımcı olmak isteyen insanlar eşzamanlı olarak wifi şireleri bir portal aracılığı ile KVKK'ya uygun olarak paylaşılacaktır. O bölgede kalan herkesten bu daha önceden verilen isimler paylaşılması oldukça kolay olacaktır.

Buradaki amacımız deprem altında telefon kullanımı olmayan isanlara hızlıca bir port açmak olacaktır. üstelik aynı wifi ile hotspot adı verilen arayüzle hızlıca bağlanan kişiyi yayımcı tarafından gösterebilmektedir. tüm bunlar bir drone üzerine monte edilerek deprem bölgesi üzerinde gezdirilip hızlı bir filtreleme ve haritalandırma yapmak oldukça mümkündür. Bu teknoloji eşzamanlı olarak başka wifi adresleri alıp bunlarla yayım haline geçebilir ve üzerine çektiği kullanıcılara hızlıca ulaşabilir. Deprem bölgelerindeki kurulan mobil baz istasyonlarındaki kargaşa eklenme gerçekleşse bile bu varlığı kişinin arama yapmaması veya arama alamaması ile eşzamanlıdır. Tüm bunlara ek olarak akıllı saatlere wireshark ile suni bildirim atılabilir.

genel bir özetle insanlardan hızlıca bir çok wifi adı toplanıp kahve işletmelerinin klon benzeri wifi alanları oluşturup insanların o wifi ağına katılmasıdır. Buradaki network istekleri buradan gözlemleyip canlılık belirtisi garantilenebilir. Debris heartbeat adı tamamen benim tarafımdan uydurulmuştur. Bir deprem ülkesi olarak buna benzer çözümler almamız gerektiğine inanıyorum.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz ilgili ekibe iletilmiştir.
